src/extensions/default/InlineBorderRadiusEditor/BorderRadiusEditorTemplate.html

```diff
@@ -117,5 +122,32 @@
                </td>
            </tr>
        </table>
-    <div>
+    </div>
+ {{/allCornerArea}}  
+
+ {{#singleCornerArea}}
```

Comment:
let's not do this! Almost all of the code in this section is the same as the `allCornerArea` section. So, just use the `allCornerArea` section and conditionally render the icon (which is the only difference). See the last part of the third point in my comment here: https://github.com/mozilla/brackets/pull/920#discussion_r164550664